### PR TITLE
[Validator] Add German translation for `WordCount` constraint

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

Needs
* https://github.com/symfony/symfony/pull/57716

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

**Review by xabbuh — CHANGES_REQUESTED** [review]
#57716 got some changes regarding pluralisation: https://github.com/symfony/symfony/pull/57716#issuecomment-2227842252

**Comment by alexandre-daubois** [comment]
Thanks for this PR!

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf [hunk 1]
```diff
@@ -442,6 +442,14 @@
...
                 <target>Dieser URL fehlt eine Top-Level-Domain.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least {{ min }} words.</source>
+                <target>Dieser Wert ist zu kurz. Er muss mindestens {{ min }} Wörter lang sein.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>This value is too long. It should contain {{ max }} words or less.</source>
+                <target>Dieser Wert ist zu lang. Er darf maximal {{ max }} Wörter lang sein.</target>
+            </trans-unit>
```
Comment (OskarStark) [inline src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]:
```suggestion
            <trans-unit id="114">
                <source>This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</source>
                <target>Dieser Wert ist zu kurz. Er muss mindestens ein Wort lang sein.|Dieser Wert ist zu kurz. Er muss mindestens {{ min }} Wörter lang sein.</target>
            </trans-unit>
            <trans-unit id="115">
                <source>This value is too long. It should contain one word.|This value is too long. It should contain {{ max }} words or less.</source>
                <target>Dieser Wert ist zu lang. Er darf maximal ein Wort lang sein.|Dieser Wert ist zu lang. Er darf maximal {{ max }} Wörter lang sein.</target>
            </trans-unit>
```

**Comment by OskarStark** [comment]
Updated

@xabbuh I don't like the "lang sein." translation, WDYT?

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf [hunk 2]
```diff
@@ -442,6 +442,14 @@
                 <source>This URL is missing a top-level domain.</source>
                 <target>Dieser URL fehlt eine Top-Level-Domain.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</source>
+                <target>Dieser Wert ist zu kurz. Er muss mindestens ein Wort lang sein.|Dieser Wert ist zu kurz. Er muss mindestens {{ min }} Wörter lang sein.</target>
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]:
```suggestion
                <target>Dieser Wert ist zu kurz. Er muss aus mindestens einem Wort bestehen.|Dieser Wert ist zu kurz. Er muss mindestens {{ min }} Wörter enthalten.</target>
```

**Comment by xabbuh** [comment]
I have suggested something for the "min message". If you agree with it, the other message can probably be adapted accordingly.

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf [hunk 3]
```diff
@@ -442,6 +442,14 @@
                 <source>This URL is missing a top-level domain.</source>
                 <target>Dieser URL fehlt eine Top-Level-Domain.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</source>
+                <target>Dieser Wert ist zu kurz. Er muss aus mindestens einem Wort bestehen.|Dieser Wert ist zu kurz. Er muss mindestens {{ min }} Wörter enthalten.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>This value is too long. It should contain one word.|This value is too long. It should contain {{ max }} words or less.</source>
+                <target>Dieser Wert ist zu lang. Er darf maximal ein Wort lang sein.|Dieser Wert ist zu lang. Er darf maximal {{ max }} Wörter lang sein.</target>
```
Comment (OskarStark) [inline src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]:
```suggestion
                <target>Dieser Wert ist zu lang. Er darf maximal aus einem Wort bestehen.|Dieser Wert ist zu lang. Er darf maximal {{ max }} Wörter enthalten.</target>
```

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
👍 but we should probably merge #57752 first

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**